pkg/features/feature.go

```diff
@@ -38,3 +40,7 @@ func (f *feature) Enabled() bool {
 		return f.defaultValue
 	}
 }
+
+func (f *feature) LoggingContext() interface{} {
```

Comment:
I am reviewing this and all looks good, except that I cannot find here any context - as known from the `context` pkg. What was the reason for naming it like this?

Reply:
Structured data in logging is also called context. But I can change the name to avoid confusion with golang context.